src/main/OPBuilder.ts

```diff
@@ -119,20 +117,12 @@ export default class OPBuilder {
     }
   }
 
-  withSupportedVersions(supportedVersions: Array<string | SupportedVersion>): OPBuilder {
+  withSupportedVersions(supportedVersion: SupportedVersion[] | SupportedVersion): OPBuilder {
     this.initSupportedVersions();
-    for (const supportedVersion of supportedVersions) {
-      this.addSupportedVersion(supportedVersion);
-    }
-    return this;
-  }
-
-  addSupportedVersion(supportedVersion: string | SupportedVersion): OPBuilder {
```

Comment:
Why is the add method of a builder removed?

with and add have different purposes

Reply:
Fixed!